### DISCUSSION TITLE
Does the boilerplate template work with existing virtual directories

### DISCUSSION THREAD

**Discussion by RattleRattler** [Dotnet-Boxed/Templates#28] [Q&A]
Good Day,

Great template. Just what I needed, especially for the security implementation. My question: Is there a way to set the template in such a way that it would not interfere with existing virtual directories on a webite?

Current: http://www.site1.com/about = this would take you to the about us page of the website,
But if http://www.site1.com/admin = is the link of the admin module hosted in a virtual directory, (and not a sub domain...preference is virtual directory), the boilerplate template looks for a route/controller named "admin" and when it doesnt find it it throws an error.

Its not really a train smash at the moment because I can easily just remote into the server myself and fix what I need fixed. But I would like to launch the http://www.site1.com/client which would be a client portal

Any help?

**Comment by RehanSaeed** [comment]
@RattleRattler Good question. I've not thought about that at all and I must admit I have not got any experience using virtual directories in MVC.

What I do know is that you can add an IgnoreRoute for your virtual directory (http://www.example.com/admin). That may solve the problem you are having. Can you explain more about where the error is coming from?

**Comment by RehanSaeed** [comment]
Closing this, let me know if you have any questions and want to re-open.
